lib/Dancer/Plugin/SimpleCRUD.pm

```diff
@@ -838,7 +838,8 @@ sub _create_add_edit_route {
             }
             $constrain_values{$name} = \%possible_values;
 
-        } elsif (my $values_from_db = $field->{mysql_values}) {
+        } elsif (my $values_from_db = $field->{mysql_values}
+                                   // $field->{mariadb_values}) {
```

Comment:
I think the // construct isn't legal in 5.8?

Reply:
Thanks for review. I can change to ||, as I can't imagine how a correct and useful value would ever evaluate to false (and thus fall back on the 'incorrect' value). I'm unable to find a machine older than 5.14, and had honestly forgotten a time when // didn't exist, but I've never wanted to be the one to cause lack of support, as I'm quite a dinosaur myself.

Is that what's causing the travis-ci problems? I can't see the travis-ci results in my dinosaur browser...

Reply:
Yeah I think changing it to || makes sense.